airbyte-integrations/connectors/source-looker/unit_tests/test_URL.py

```diff
@@ -0,0 +1,44 @@
+import re
+import unittest
+
+def valid_URL(test_str):
+    pattern="(^[a-zA-Z0-9._-]*\.looker\.com$)|(^(?:[a-zA-Z0-9._-]*)?[a-zA-Z0-9._-]*\.[a-zA-Z0-9._-]*(?::\d{1,5})?$)|(^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(?::\d{1,5})?$)"
+    result = re.match(pattern, test_str)
+    if result is not None:
+        return result.group(0)
+
+
+
+class TestURLPattern(unittest.TestCase):

```

Comment:
also we are trying to follow pytest-style tests, please check [this](https://docs.pytest.org/en/6.2.x/example/parametrize.html#different-options-for-test-ids) for example how you can do it